bolt/internal/internal.proto

```diff
@@ -126,6 +126,8 @@ message AlertRule {
 message User {
 	uint64 ID               = 1; // ID is the unique ID of this user
 	string Name             = 2; // Name is the user's login name
+	string Provider         = 3; // Provider is the provider that certifies and issues this user's authentication, e.g. GitHub
+  string Scheme           = 4; // Scheme is the scheme used to perform this user's authentication, e.g. OAuth2 or LDAP
```

Comment:
Would you indent this the same level as the other fields?  Sometimes it is hard to see in an editor if it is a tab or spaces.